usaspending_api/common/api_request_utils.py

```diff
@@ -429,6 +430,14 @@ def build_response(self):
         elif scope == "domestic":
             scope_q = Q(**{"location_country_code": "USA"})
 
+        usage_q = Q()
+        if usage == "recipient":
+            usage_q = Q(recipient_flag=True)
+        elif usage == "place_of_performance":
+            usage_q = Q(place_of_performance_flag=True)
+        elif usage == "both":
```

Comment:
"all" is when you don't care about if it is pop or recipient, both is if you want the case where both are true. Slightly different use case. "all" has no filters, so it doesn't need to have a case here.